src/main/java/com/minecolonies/coremod/event/ClientEventHandler.java

```diff
@@ -67,11 +66,13 @@ public void renderWorldLastEvent(@NotNull final RenderWorldLastEvent event)
             final BlockPos position = Settings.instance.getPosition();
             if (Settings.instance.getStructureName().contains(AbstractEntityAIStructure.WAYPOINT_STRING))
             {
-                RenderUtils.renderWayPoints(position, world, event.getPartialTicks());
+                //TODO: Implement waypoint rendering.
+                //RenderUtils.renderWayPoints(position, world, event.getPartialTicks());
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3ACommentedOutCodeLine)